packages/serverless-contracts/package.json

```diff
@@ -32,7 +34,7 @@
     "package-types": "tsc -p tsconfig.build.json",
     "package-types-aliases": "tsc-alias -p tsconfig.build.json",
     "test": "nx run test-linter && nx run test-type && nx run test-unit && nx run test-circular",
-    "test-circular": "pnpm depcruise --validate .dependency-cruiser.js .",
+    "test-circular": "pnpm depcruise --config -- .",
```

Comment:
Why was it necessary to remove the `--validate` option?

Reply:
It is explained here: https://github.com/sverweij/dependency-cruiser/blob/develop/doc/cli.md#--config---validate

Basically, `--validate` and `--config` are aliases, and we need to use it to have the default config (in our case `.dependency-cruiser.cjs`) loaded

Reply:
Perfect!